e2e/backend_service_update_test.go

```diff
@@ -226,3 +226,39 @@ func TestBackendServiceUpdate(t *testing.T) {
 	wantBS = &compute.BackendService{LoadBalancingScheme: "INTERNAL_MANAGED"}
 	checkBackendService(t, ctx, theCloud, bs1ID, wantBS, compareLBScheme)
 }
+
+func TestEnsureBackendService(t *testing.T) {
```

Comment:
ensure test should: 
1. create all resources 
2. call planing again on the same graph and verify that planning doesn't return action to perform (that means that GCE state is equal desired state)

Reply:
Please take another look.